packages/@uppy/companion/test/__mocks__/purest.js

```diff
@@ -29,6 +33,11 @@ class MockPurest {
       const responses = fixtures[this.opts.providerName].responses
       const url = this._query ? `${this._requestUrl}?${this._query}` : this._requestUrl
       const endpointResponses = responses[url] || responses[this._requestUrl]
+      if (endpointResponses == null || !has(endpointResponses, this._method)) {
+        done(new Error(`No fixture for ${this._method} ${url}`))
+        return
+      }
```

Comment:
For debugging: a test failure on CI was due to a missing fixture and it was not super easy to find